Polyhedron/demo/Polyhedron/Plugins/IO/Surface_mesh_io_plugin.cpp

```diff
@@ -0,0 +1,91 @@
+#include <CGAL/Three/Polyhedron_demo_io_plugin_interface.h>
+#include <CGAL/Three/Polyhedron_demo_plugin_interface.h>
+#include <fstream>
+
+#include <QApplication>
+#include <QObject>
+#include <QAction>
+#include <QMainWindow>
+#include <QDebug>
+#include "Scene_surface_mesh_item.h"
+#include "Scene_polygon_soup_item.h"
+#include "Messages_interface.h"
+
```

Comment:
Messages_interface.h is needed to use messages_interface, which I do to notify the user when a file cannot be open or when the faces are not oriented correctly and a polygon_soup is loaded instead of a surface_mesh.